data/hai/hai reveal 4 middle.txt

```diff
@@ -1295,23 +1295,23 @@ mission "Hai Reveal [A13] A New Elder"
-			`	He manages to make that last line sound both bewildered and grumpy at the same time. "That Teeneep you've been running around with, and yes don't look at me in shock; Sayari told me when we were concerned you might not get out. You've developed some friends in high places and I admire that... where was I?"`
+			`	He manages to make that last line sound both bewildered and grumpy at the same time. "That Teeneep you've been running around with - and yes, don't look at me in shock, Sayari told me when we were concerned you might not get out. You've developed some friends in high places and I admire that... where was I?"`
 			`	"Teeneep!" says Alondo, perhaps a little too loudly, nearly poking Remington in the face.`
-			`	"Of course, that Teeneep of yours had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot, at least now that sir pitchforks and pulse rifles down there has definitively lost. She's a firebrand is that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech."`
+			`	"Of course, that Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot, at least now that sir pitchforks and pulse rifles down there has definitively lost. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech."`
 			choice
 				`	(Wait with them.)`
 				`	(Drink with them.)`
-			`	As it turns out, Teeneep's speech is in Hai with no translation available. It's short, punchy, and restrained though; or at least her delivery of it has that vibe. With nothing else to do and it being well into the evening, there's nothing for it but to call it a night.`
+			`	As it turns out, Teeneep's speech is in Hai with no translation available. It's short, punchy, but restrained; or at least her delivery of it had that feeling. With nothing else to do, and it being well into the evening, you call it a night.`
```

Comment:
```suggestion
			`	As it turns out, Teeneep's speech is in Hai with no translation available. It's short and punchy, but restrained; or at least her delivery of it has that feeling. With nothing else to do, and it being well into the evening, you call it a night.`
```

Transition in tense from "is" in "it's" to "had" is no good.

Reply:
I had thought that the player was looking back on it as it happened before, because the player is calling it a night right now. But I would have had to change the description above too, and I didn't.